samples/react-native/e2e/starter.test.ts

```diff
@@ -0,0 +1,12 @@
+import { describe, it, beforeAll } from '@jest/globals';
+import { device, expect } from 'detox';
+
+describe('Shows HomeScreen', () => {
+  beforeAll(async () => {
+    await device.launchApp();
+  });
+
+  it('Shows Bottom Tab Bar', async () => {
+    await expect(element(by.text('Performance'))).toBeVisible();
+  });
```

Comment:
What are we testing here, exactly, that the sample app launched?
Will this pass if there is an error right after the screen is rendered?